sudo/omnect-device-service

```diff
@@ -5,5 +5,5 @@ omnect_device_service ALL=(omnect) NOPASSWD: /usr/bin/tee /home/omnect/.ssh/auth
 omnect_device_service ALL=(root) NOPASSWD: /bin/journalctl --sync
 
 # establish ssh tunnel
-Cmnd_Alias SSH = /usr/bin/ssh, /usr/bin/ssh-keygen
+Cmnd_Alias SSH = /usr/bin/ssh, /usr/bin/ssh-keygen, /bin/cat, /bin/rm, /usr/bin/tee
```

Comment:
are there any dynamic filenames you have to handle? otherwise you could restrict the `sudo` commands of `ssh_tunnel_user` even more.

Reply:
That's exactly the issue: I want to cat/rm/tee files in /run/omnect-device-service/ssh-tunnel but their actual names vary.

Reply:
https://manpages.ubuntu.com/manpages/lunar/en/man5/sudoers.5.html Chroot_Spec looks as if it could possible help, though.